tritter/server/server.go

```diff
@@ -10,8 +10,8 @@ import (
 	"google.golang.org/grpc"
 )
 
-const (
-	listenAddr = "localhost:50051"
+var (
+	listenAddr = flag.String("listen_addr", "localhost:50051", "the address the server is listening on")
```

Comment:
LGTM, but I'm pretty sure you'll need to change all instances of `listenAddr` to `*listenAddr` to get the flag value instead of the flag pointer.

Reply:
Thanks. Updated all instances of `listenAddr` to `*listenAddr`